docs/reference/semaphore-changelog.md

```diff
@@ -8,6 +8,13 @@ Thank you for using Semaphore!
 We continuously deploy changes that improve the product for you.
 This page is updated on a weekly basis.
 
+### Week of August 17, 2020
+**(New) Install-package command**  
+"Toolbox" command line tools have been extended to include `install-package` utility.  
+The `install-package` command can help with package installations by automatically caching the desired packages and their dependencies.  
+
+To find out more, please check [install-package](https://docs.semaphoreci.com/reference/toolbox-reference/#install-package) documentation page.
```

Comment:
missing 'the'
=> To find out more, please check the [install-package](https://docs.semaphoreci.com/reference/toolbox-reference/#install-package) documentation page.